src/adapter/src/coord/command_handler.rs

```diff
@@ -180,6 +180,17 @@ impl Coordinator {
             }
             let plan = CreateRolePlan {
                 name: session.user().name.to_string(),
+                attributes: RoleAttributes {
+                    // TODO(jkosh44) We need to start checking if the frontegg user is an admin.
```

Comment:
We should not merge this until we check these roles, because then we'll have roles persisted to the stash with incorrect superuser attributes.

Reply:
Since we have no way of knowing which roles were created with `CREATE ROLE <name> LOGIN SUPERUSER` or created by logging in via frontegg, I was planning on just grandfathering all existing roles into superuser status. Then I was going to leave it up to the organizations to revoke superuser from non-admins. Considering that, this isn't much different from the existing behavior of all new roles are given superuser status.

The other option would be to to retroactively revoke superuser status from everyone, but then there'd be no superusers left to grant it back to others.

Reply:
Why do we need to allow users to type `CREATE ROLE <name> LOGIN` at all? They'll never be able to specify anything that passes frontegg auth. It might actually create problems because we usually only create users on first connection.

Reply:
We don't strictly need it, but this would allow organizations to set a role up with the proper permissions before that user signs in for the first time. Otherwise they can't give a user permissions until they log in for the first time.

Also, it seems like a bit of unnecessary coupling with frontegg to prevent creating login users via SQL.

Reply:
What would happen if the admin granting privs forgot to add the LOGIN attribute, then that user wouldn't be able to login until they got granted that by a separate person? It's also a way of denying LOGIN to specific users that don't exist yet? Kinda weird.

Reply:
I guess this is somewhat related to the question on the design doc about syncing `SUPERUSER` status with frontegg roles. 
If a user tries to login via Frontegg, and we see that the role already exists without `LOGIN`, then do we update the role to contain `LOGIN` (i.e. sync the Materialize role with the Frontegg role)?

I think I'm actually OK with doing some Frontegg-Materialize syncing when a new connection starts up. I'm just worried about having an RBAC implementation that relies on our current frontegg usage to be correct.